pkg/network/vmicontroller/status.go

```diff
@@ -51,6 +51,16 @@ func UpdateStatus(clusterConfig clusterConfigChecker, vmi *v1.VirtualMachineInst
 
 	interfaceStatuses = append(interfaceStatuses, secondaryIfaceStatuses...)
 
+	// virt-handler adds status for interfaces only existing inside the guest that were discovered by QEMU GuestAgent
+	guestInternalIfaceStatuses := vmispec.FilterInterfaceStatuses(
```

Comment:
Might be misunderstanding, but do we need to account for the case where the `InfoSource` is `InfoSourceDomainAndGA` in the case of older virt-launchers?

Reply:
Changed the filter condition to account for this scenario.